### PR TITLE
local services.yml is the same for every project so we can safly incl…

### PR DESCRIPTION
**Pull request by rainer-friederich**
…ude this. Services differ for some projects so we check for the project service file. New version will use override.services.yml

### CODE REVIEW

File: src/site.settings.php [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
+
+// Load service overrides
+// @DEPRECATED
+if(file_exists($app_root . '/' . $site_path . '/services.yml')) {
+  $settings['container_yamls'][] = $app_root . '/' . $site_path . '/services.yml';
+}
+
+// Load service overrides
+if(file_exists($app_root . '/' . $site_path . '/override.services.yml')) {
+  $settings['container_yamls'][] = $app_root . '/' . $site_path . '/override.services.yml';
```
Comment (tassilogroeper) [inline src/site.settings.php]: 1. Do we really want to introduce a new naming here? `override` vs. `local`?
2. Please add documentation to readme.

File: src/site.settings.php [hunk 2]
```diff
@@ -1,16 +1,28 @@
...
 use wearewondrous\PshToolbelt\SiteSettings;
 
 $databases = [];
 $config_directories = [];
-$settings['container_yamls'][] = $app_root . '/' . $site_path . '/services.yml';
+
+$settings['container_yamls'][] = '../services/default.services.yml';
+
+// Load service overrides
+// @DEPRECATED
```
Comment (tassilogroeper) [inline src/site.settings.php]: Why deprecated?

File: src/site.settings.php [hunk 3]
```diff
@@ -1,16 +1,28 @@
...
 
-use Platformsh\ConfigReader\Config;
+use Platformsh\ConfigReader\Config as PlatformshConfig;
 use wearewondrous\PshToolbelt\SiteSettings;
 
 $databases = [];
 $config_directories = [];
-$settings['container_yamls'][] = $app_root . '/' . $site_path . '/services.yml';
+
+$settings['container_yamls'][] = '../services/default.services.yml';
```
Comment (tassilogroeper) [inline src/site.settings.php:16]: Should we suport, multiple drupal versions, if this would change?

**Review by tassilogroeper — CHANGES_REQUESTED** [review]

File: src/site.settings.php (hunk 1 above)
Comment (rainer-friederich) [reply src/site.settings.php]: I always thought of local as only for local development but local as project local makes sense!

File: src/site.settings.php (hunk 3 above)
Comment (rainer-friederich) [reply src/site.settings.php:16]: We could do this with semver if we would require drupal core as dependency with a minimum version

Comment (tassilogroeper) [reply src/site.settings.php:16]: Use the `DRUPAL_VERSION` global to place version dependent includes there? For now, it is fine of course

File: src/site.settings.php (hunk 1 above)
Comment (tassilogroeper) [reply src/site.settings.php]: Can you please add a new section to the documentation, how to use this, and whats the logic for that?

**Pull request merged**